crates/common/src/prompt/confirm.rs

```diff
@@ -0,0 +1,25 @@
+use std::fmt::Display;
```

Comment:
Also not used

Reply:
Yeah, we're not using this menu yet. This could be helpful if we have an optional step. Cloning some dep or something like that. I'd keep it for now